### DISCUSSION TITLE
Custom url for reading stream with specific scheme like rtsp?

### DISCUSSION THREAD

**Discussion by u-Kotovsky** [bluenviron/mediamtx#3616] [Questions] [answered]
### Question

Example:
Server does reads rtsp://localhost:8554/mystream
But I want it to read something different like myscheme://localhost:8554/mystream so it will be the same as rtsp one in result

**Comment by aler9** [comment, marked as answer]
The question is not clear.
The README lists all supported ways and protocols to read streams from the server.
If you still have troubles, open another question and provide precise details on what you want to accomplish.
